torchreid/models/common.py

```diff
@@ -28,6 +28,7 @@ def __init__(self,
           self.classification = classification
           self.contrastive = contrastive
           self.pretrained = pretrained
+          self.classification_classes = {}
```

Comment:
Can't find any initialization code for this field.

Reply:
See `load_pretrained_weights`

Reply:
Classes on evaluation are checked only if we have a reference in checkpoint.

Reply:
But who is responsible to pass `classes_map` into `checkpoint`?

Reply:
`Engine.save_model` will do this if possible

Reply:
I see, Ok.